readme.md

```diff
@@ -6,185 +6,165 @@
+Next come the **Flags**. Think of these as "options" that tell the command how to do what it's about to do. There may be zero or more options.
+* Sometimes you won't be using any options. Other times just one or maybe even more!
+* Options usually start with one or two dashes. If the option is a letter, then one dash (e.g., `-a`). If it's a whole word, then two dashes (e.g., `--all`).
 
-http://catb.org/esr/writings/taoup/html/ch10s05.html#id2948149
+> [Here's a list](http://catb.org/esr/writings/taoup/html/ch10s05.html#id2948149) of some single-letter flags you might encounter. This is not an exhaustive list.
 
-Finally come the arguments. These are what you want to do the action to. Usually
-these are file names, but they could be URLs, or other things.
+Finally come the **Arguments**. These are "targets", or what you want to do the action to. These could be file names, URLs, etc.
 
 #### Common Patterns
 
-The commands given to a CLI shell are often in one of the following forms:
+The commands entered into the CLI are often in one of the following forms..
 
 - `doSomething how toFiles`
 - `doSomething how sourceFile destinationFile`
 
-Where *doSomething* is, in effect, a verb, *how* an adverb (for example, should the command be executed "verbosely" or "quietly") and *toFiles* an object or objects (typically one or more files) on which the command should act.
+Where **doSomething** is, in effect, a verb, **how** an adverb (for example, should the command be executed "verbosely" or "quietly") and **toFiles** an object or objects (typically one or more files) on which the command should act.
 
-Let's take a look at something we did for installfest.
+Let's take a look at something we did for Installfest...
 
 ```
 $ brew install git
 ```
 
-> When we type this command and hit enter , we're saying, "Computer, we're about to do something with homebrew. The thing were going to do is install something. What we want to do is install it.
+When we type this command and hit enter, we're saying, "Computer, we're about to do something with Homebrew. The thing were going to do is install something. What we want to do is install `git`.
 
-It should be noted that all commands follow this pattern, but many do.
+> Not all commands follow this pattern, but many do.
 
-### Commands run in an 'environment'
+### Commands Run in an "Environment"
 
-We won't go deep into environments yet, but everytime a command is run, it may
-choose to look at what are called `environment variables`. These are essentially
-variables that are semi-permanent and can provide information or options to help
-a command do its job.
+We won't go deep into environments yet, but every time a command is run, it may choose to look at what are called "environment variables". These are semi-permanent variables that provide information or options to help a command do its job.
+<!-- AM: Any worthwhile examples? Worth mentioning this? -->
```

Comment:
in installfest, we set up EDITOR environment variable - otherwise git would use vim